### DISCUSSION TITLE
vs code安裝虛擬程式問題..

### DISCUSSION THREAD

**Discussion by Rebecca111419** [1101-GS4719-C/Python-Community-GS4719-C#1714] [Q&A]
剛才照老師上課步驟安裝虛擬程式，但它無法從venv中顯示，之後無法打開網站，不知該如何解決.... 看不懂它的解釋..
<img width="676" alt="螢幕擷取畫面 2023-11-22 172537" src="https://github.com/NCU-GS4719-Python/Python-Community/assets/144880481/13ee7ad6-38bc-4e7c-9513-27303c09f277">

**Comment by lazywulf** [comment]
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-7.3
感覺你那是被系統擋爛。你看看這連結好了，搞不好會幫助到你。
如果真是這個問題也不要罵你的系統，他很努力在保護你。不明 .ps1 本來就很危險 (對系統而言)。
